sql/snowflake/SnowflakeParser.g4

```diff
@@ -256,63 +245,83 @@ execute_task
     ;
 
 explain
-    : EXPLAIN ( USING (TABULAR | JSON | TEXT) )? sql_command
+    : EXPLAIN (USING (TABULAR | JSON | TEXT))? sql_command
     ;
 
 parallel
     : PARALLEL EQ num
     ;
 
 get_dml
-    : GET internal_stage FILE_PATH
-            parallel?
-            pattern?
+    : GET internal_stage FILE_PATH parallel? pattern?
     ;
 
 grant_ownership
-    : GRANT OWNERSHIP
-         ( ON ( object_type_name object_name | ALL object_type_plural IN ( DATABASE id_ | SCHEMA schema_name ) )
-         | ON FUTURE object_type_plural IN ( DATABASE id_ | SCHEMA schema_name )
-         )
-         TO ROLE id_ ( ( REVOKE | COPY ) CURRENT GRANTS )?
+    : GRANT OWNERSHIP (
+        ON (
+            object_type_name object_name
+            | ALL object_type_plural IN ( DATABASE id_ | SCHEMA schema_name)
```

Comment:
@mike-lischke any reason why the space characters after `( DATABASE` were kept?

Reply:
Well, no specific reason actually. Might be a formatter bug. I believe that's coming up when an alt is placed entirely on one line and contains groups.

Reply:
sry, space character after `(` and before `DATABASE`